dashboard/test/controllers/followers_controller_test.rb

```diff
@@ -315,4 +338,15 @@ class FollowersControllerTest < ActionController::TestCase
 
     assert_redirected_to admin_directory_path
   end
+
+  # Place test at bottom to avoid reCAPTCHA issues with pre-existing tests
+  test 'student_user_new displays error when joining 3 or more sections in 24 hours without completing captcha' do
+    Recaptcha.configuration.skip_verify_env.delete("test")
```

Comment:
Is there a way to do this without modifying external state (e.g. via mocking)?  If not, let's at least revert this change at the end of the test.  (In general, unit tests should not modify external state.)

Reply:
Agreed - this felt like a hack that would come back to bite later one. Not sure how we would be able to mock it since the `verify_recaptcha` method call comes directly from the recaptcha gem we use. For now, I will just revert the change after running the test. Please let me know if you have other ideas!